MarsQA/StepDefinitions/LanguagesStepDefinitions.cs

```diff
@@ -0,0 +1,56 @@
+using MarsQA.Drivers;
+using MarsQA.SpecflowPages;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using System;
+using System.Threading;
+using TechTalk.SpecFlow;
+
+namespace MarsQA
+{
+    [Binding]
+    public class LanguagesStepDefinition 
+    {
+        LoginPage loginPageObj = new LoginPage();
```

Comment:
Please make use of constructors, please see 
 https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors